README.md

```diff
@@ -1,27 +1,32 @@
 <p align="center"><img src="https://user-images.githubusercontent.com/6799467/65944518-68834d80-e421-11e9-9b14-6ca26a16108a.png" width="350px"></p>
 <h1 align="center">pfetch</h1>
-<p align="center">A pretty system information tool written in POSIX sh</p><br>
+<p align="center">A minimal yet pretty system profiler written in POSIX sh</p><br>
```

Comment:
Don't think profiler is the right word here. It's usually used in context of measuring some value like perfomance or complexity

Reply:
I see, still wondering whether there might be a better word - to call it a "tool" is almost an overstatement, because for me that implies versatility, while this is more of a helper script.